src/ansys/fluent/core/solver/flobject.py

```diff
@@ -372,17 +388,49 @@ class String(SettingsBase[str], Textual):
     _state_type = str
 
 
-class Filename(SettingsBase[str], Textual):
+class InputOutputFileBase:
```

Comment:
I haven't looked at the other changes in this file, but the design already looks different to what we discussed. It's unclear how this class would fit into a rationale design.

Reply:
As per our latest discussion, we have decided to get things working first and not to use mixins.